### PR TITLE
Feature #563: Schema updates for admin site.

### PR DESCRIPTION
**Pull request by trigramdev9**
Porting over from NFT to support api token blocking in the admin site: https://github.com/nftstorage/nft.storage/pull/1203

### CODE REVIEW

File: packages/db/postgres/tables.sql [hunk 1]
```diff
@@ -260,3 +280,19 @@ CREATE TABLE IF NOT EXISTS pinning_authorization
   inserted_at     TIMESTAMP WITH TIME ZONE DEFAULT timezone('utc'::text, now()) NOT NULL,
   deleted_at      TIMESTAMP WITH TIME ZONE
 )
```
Comment (vasco-santos) [inline packages/db/postgres/tables.sql:282]:
```suggestion
);
```

**Review by vasco-santos — CHANGES_REQUESTED** [review]
Thanks for the PR @trigramdev9 

It was missing a `;` in the previous SQL statement that is breaking CI

File: packages/db/postgres/tables.sql [hunk 2]
```diff
@@ -260,3 +280,19 @@ CREATE TABLE IF NOT EXISTS pinning_authorization
...
+
+CREATE VIEW admin_search as
+select
+  u.id::text as user_id,
+  u.github_id as username,
```
Comment (vasco-santos) [inline packages/db/postgres/tables.sql:287]:
```suggestion
  u.github as username,
```

Not sure if this is what you want, just to make sure. user.github is a numeric github ID, not the handle. We don't track the handle in web3.storage

File: packages/db/postgres/tables.sql (hunk 1 above)
Comment (trigramdev9) [reply packages/db/postgres/tables.sql]: woops.  my bad!

File: packages/db/postgres/tables.sql (hunk 2 above)
Comment (trigramdev9) [reply packages/db/postgres/tables.sql]: removed this field.

**Review by vasco-santos — APPROVED** [review]
LGTM

**Pull request merged**